index.js

```diff
@@ -23,3 +23,4 @@ export { default as FocusManager } from './components/FocusManager/index.js';
 export { default as Icon } from './components/Icon/index.js';
 export { default as MarqueeText } from './components/MarqueeText/index.js';
 export { default as Row } from './components/Row/index.js';
+export { default as Base } from './components/Base';
```

Comment:
I'd like to mimic the inner source repo here and create folders for elements vs layout vs mixins, then have each of those folders with their own index.js to handle the exports in that folder.

Reply:
is ok if we tackle splitting those out in a separate PR? I'm working on it in my layout updates